src/pages/docs/platform/auth/capabilities.mdx

```diff
@@ -0,0 +1,897 @@
+---
+title: Identified clients and capabilities
```

Comment:
I would not combine these two pages (they are separate in the existing docs) it's a stretch trying to combine the two pretty distict concepts..